src/accordion/accordion.spec.ts

```diff
@@ -322,6 +322,21 @@ describe('ngb-accordion', () => {
     });
   });
 
+  it('should have the appropriate CSS collapsed classes', () => {
```

Comment:
Couldn't we add assert to https://github.com/ExFlo/ng-boostrap/blob/e1064feca33b001b9cad323edb4f5ffbb0c0373a/src/accordion/accordion.spec.ts#L37 instead of writing whole new test?

The advantage is that it would be verified with all the existing tests.

Reply:
@ExFlo I was more thinking of sth like:

```
panelsTitles.map((titleEl: HTMLButtonElement) => {
   const isAriaExpanded = titleEl.getAttribute('aria-expanded') === 'true';
   const isCSSCollapsed = titleEl.classList.contains('collapsed');
   return isAriaExpanded === !isCSSCollapsed ? isAriaExpanded : fail('inconsistent state');
});
```

not well thought-through, just a rough idea...

Reply:
So we just test that collapsed is correctly removed, but not that it is in place the first time right?

Reply:
During the first time rendering we should have:
* `isAriaExpanded=false`
* `isCSSCollapsed=true`

Given this the `isAriaExpanded === !isCSSCollapsed` condition should be `true` and the return value `false`.

If the class wouldn't be here initially we would get:
* `isAriaExpanded=false`
* `isCSSCollapsed=false`

and the `isAriaExpanded === !isCSSCollapsed` test would fail breaking the test. So I believe that the case of initial display would be covered as well.

But hey, this is just a rough idea, feel free to adapt / discard.

Reply:
right I missed the use case of the function. Thx